clustering/singleton/api/src/main/java/org/wildfly/clustering/singleton/election/PreferredSingletonElectionPolicy.java

```diff
@@ -15,7 +15,9 @@
  * if the preferred node is not a candidate.  The means of specifying the preferred node is
  * the responsibility of the extending class.
  * @author Paul Ferraro
+ * @deprecated Replaced by {@link org.wildfly.clustering.singleton.election.SingletonElectionPolicy#prefer(java.util.function.Predicate)}.
```

Comment:
The prefer(..) takes a list of predicates, so this link won't resolve, it should be:

`{@link org.wildfly.clustering.singleton.election.SingletonElectionPolicy#prefer(List)}`